microsoft/PowerToys
# updating to agent that supports .net 6

## Summary of the Pull Request

**What is this about:**
Updating CI and projects for .net framework 6. This means we have to update to Visual Studio 2022 as well, since Visual Studio 2019 doesn't support .net framework 6.

**What is included in the PR:** 
- [x] update everything to .NET 6
- [x] Update installer to include .NET 6 runtime
- [x] Update installer to remove .NET 5 runtime
- [x] remove .net 5 reference from CI.yml for unit tests at start
- [x] Fix Windows Terminal PT Run Plugin unit test max file length issue (add back in unit test)
- [x] Fix PowerToys Run issue when disabling the module in .net 6 Framework
- [x] Update some docs for Visual Studio 2022.

**How does someone test / validate:** 

Build it with Visual Studio 2022, install it and test everything a bit. See if it works, please report any issues you find.
You'll need https://marketplace.visualstudio.com/items?itemName=WixToolset.WixToolsetVisualStudio2022Extension to build the installer with Visual Studio 2022 as well.

## Quality Checklist

- [x] **Linked issue:** #16088
